src/aws/flb_aws_util.c

```diff
@@ -40,7 +40,7 @@
 #define S3_KEY_SIZE 1024
 #define RANDOM_STRING "$UUID"
 #define INDEX_STRING "$INDEX"
-#define MAX_RESP_BUFFER_SIZE 0
+#define MAX_RESP_BUFFER_SIZE 0 /* 0 means unlimited capacity as per requirement */
```

Comment:
Nit: can we rename this to `MAX_AWS_RESP_BUFFER_SIZE` to avoid header conflicts and make it clear that the value is tailored for AWS service responses? Maybe even include `FLB_` at the beginning?